src/components/ui/button.tsx

```diff
@@ -30,6 +30,7 @@ const buttonVariants = cva(
         xs: "h-6 rounded-md px-2 text-xs",
         sm: "h-8 rounded-md px-3 text-xs",
         lg: "h-10 rounded-md px-8",
+        xl: "h-10.8 px-4 py-2",
```

Comment:
Lets nor extend shadcn buttons

Reply:
I have now removed the "xl" size and have directly added the necessary height and padding in the className of the respective buttons.